framer/Material-UI.framerfx/package.json

```
@@ -14,9 +14,9 @@
     "url": "https://github.com/mui-org/material-ui/issues"
   },
   "dependencies": {
-    "@mui/material": "^5.0.0",
-    "@mui/icons-material": "^5.0.0",
-    "@mui/utils": "^5.0.0",
+    "@mui/icons-material": "5.0.1",
```

Careful about lerna always removing the caret. Any published package should use caret ranges 
```suggestion
    "@mui/icons-material": "^5.0.1",
```

Probably safer to just use it everywhere until we figure out why `lerna:version` removes the caret.

---

Ahhh, forgot about this, let me update.